readme.md

```diff
@@ -94,6 +115,55 @@ You may selfhost (AKA run your own instance of) this bot under the following cir
 - You understand that no support will be provided to aid you in self-hosting.
 - You agree to not submit any issues, features, or pull requests related to bugs exclusively related to self-hosting.
 
+### Configuration
+
+config.json, the oauth part is not mandatory (it is used for the web interface) but if you don't needed you need to leave the settings with default values:
+```
+{
+  "token": "bot token",
+  "oauth": {
+    "id": "client id",
+    "redirect": "http://localhost:8108/callback",
+    "secret": "client secret",
+    "authSecret": "someamazingsecret"
+  },
+  "shard": 0,
+  "prefix": "uno",
+  "shards": {
+    "max": 1,
+    "perCluster": 1
+  }
+}
+```
+
+config/config.json (support various database with this example will use SQLite):
+
+```
+{
+  "production": {
+    "username": null,
+    "password": null,
+    "database": "sqlite::memory:",
+    "host": null,
+    "dialect": "sqlite",
+    "storage": "database.sqlite"
+  },
+  "development": {
+    "username": null,
+    "password": null,
+    "database": "sqlite::memory:",
+    "host": null,
+    "dialect": "sqlite",
+    "storage": "database.sqlite"
+  }
+}
+```
+
+### Dependencies
+
+After running `npm install` it is required to migrate the new database (after put the configuration files) and run `npx sequelize-cli db:migrate`.  
```

Comment:
I'd recommend using the npm script instead, so you don't have to deal with npx.
```suggestion
After running `npm install` it is required to migrate the new database (after put the configuration files) and run `npm run sequelize db:migrate`.
```

Reply:
Additionally, we could make a new run script in package.json to run the migration command directly.